ansibullbot/utils/extractors.py

```diff
@@ -498,10 +498,6 @@ def get_module_authors(self):
         if not ydata:
             return []
 
-        # sometimes the field is 'author', sometimes it is 'authors'
-        if 'authors' in ydata:
-            ydata['author'] = ydata['authors']
-
```

Comment:
Should not the duplicated code below be updated too ? https://github.com/ansible/ansibullbot/blob/de3fde5704e113d30babaa5340dcd7f1c6b9ea7f/ansibullbot/utils/moduletools.py#L899-L902

Reply:
:+1: